docs/backup-restore.md

```diff
@@ -1,134 +1,183 @@
+However, if the quorum of cluster nodes loses their data, you will need a transactionally consistent backup (snapshot) to restore the cluster to a certain transactionally consistent point.
 
-You must create a Cosmos DB for NoSQL account with a continuous backup policy enabled to use point-in-time restore (PITR) feature. Backups are created continuously after it is enabled.
-To specify a transactionally-consistent restore point, please pause the ScalarDB application of a Cosmos DB for NoSQL as described [here](#transactionally-consistent-backups-during-pause-duration).
+To create a transactionally consistent cluster-wide backup, pause the application that is using ScalarDB or ScalarDB Cluster and take snapshots of the nodes as described in [Back up with explicit pausing](#back-up-with-explicit-pausing) or stop the Cassandra cluster, take copies of all the data in the nodes, and start the cluster.
+</div>
+<div id="Cosmos_DB_for_NoSQL2" class="tabcontent" markdown="1">
 
-**DynamoDB**
+You must create a Cosmos DB for NoSQL account with a continuous backup policy that has the PITR feature enabled. After enabling the feature, backups are created continuously.
 
-You must enable the point-in-time recovery (PITR) feature for DynamoDB tables. If you use [ScalarDB Schema Loader](schema-loader.md) for creating schema, it enables PITR feature for tables by default.
-To specify a transactionally-consistent restore point, please pause the ScalarDB application of a DynamoDB as described [here](#transactionally-consistent-backups-during-pause-duration).
+To specify a transactionally consistent restore point, pause your application that is using ScalarDB with Cosmos DB for NoSQL as described in [Back up with explicit pausing](#back-up-with-explicit-pausing).
+</div>
+<div id="DynamoDB2" class="tabcontent" markdown="1">
 
-## Restore Backup
+You must enable the PITR feature for DynamoDB tables. If you're using [ScalarDB Schema Loader](schema-loader.md) to create schemas, the tool enables the PITR feature for tables by default.
 
-### MySQL (restore)
+To specify a transactionally consistent restore point, pause your application that is using ScalarDB with DynamoDB as described in [Back up with explicit pausing](#back-up-with-explicit-pausing).
+</div>
+</div>
 
-If you use MySQL and `mysqldump` to create a backup file, use `mysql` command to restore the file as specified [in the MySQL doc](https://dev.mysql.com/doc/mysql-backup-excerpt/8.0/en/reloading-sql-format-dumps.html).
+## Restore a backup
 
-### PostgreSQL (restore)
+How you restore a transactionally consistent backup depends on the type of database that you're using. Select your database to see how to restore a transactionally consistent backup for ScalarDB.
 
-If you use PostgreSQL and `pg_dump` to create a backup file, use `psql` command to restore the file as specified in [the PostgreSQL doc](https://www.postgresql.org/docs/current/backup-dump.html#BACKUP-DUMP-RESTORE).
+<div id="tabset-3">
```

Comment:
Ditto. This doesn't cover all the databases we support. So I think we should note that these are merely examples for some of our supported databases.

Reply:
Understood. I've added a note about that here as well.